portfolio/src/main/webapp/index.html

```diff
@@ -198,9 +198,14 @@ <h1 class ="titles">Photography</h1>
       </secion>
     </div>
     <div class="java-tests">
-      <button onclick="getName()">Get my Name</button>
-      <h1 id="name"></h1>
-      <ul id="make-me"></ul>
+      <form action="/data" method="POST">
+        <p>Enter a comment</p>
+        <textarea name="comment"></textarea>
+        <input type="submit" />
```

Comment:
super nit: I don't believe we usually put a space before the />, looks odd to me anyhow